fvm/bootstrap.go

```diff
@@ -77,6 +77,7 @@ type BootstrapParams struct {
 	storagePerFlow                   cadence.UFix64
 	restrictedAccountCreationEnabled cadence.Bool
 	setupEVMEnabled                  cadence.Bool
+	evmAbiOnly                       cadence.Bool
```

Comment:
Can't quite see it, but this will/should be enabled by default, how is that achieved?

I think the idea is that ABI-only `EVM` contract will always be enabled, independent of the "full EVM functionality" in FVM?

Reply:
Updated in https://github.com/onflow/flow-go/pull/5230/commits/a21a51bffd57364b092959924172640d111d01c3